src/app/base/components/InfoIconTooltip/InfoIconTooltip.tsx

```diff
@@ -0,0 +1,13 @@
+import type { ReactNode } from "react";
+
+import { Icon, Tooltip } from "@canonical/react-components";
+
+const InfoIconTooltip = ({ message }: { message: ReactNode }) => {
+  return (
+    <Tooltip message={message} position="right">
```

Comment:
Might be worth using the TooltipButton component here, so the tooltip stays visible if you click the icon